container/root/run.d/10-nginx.sh

```diff
@@ -25,3 +29,9 @@ then
   echo "[nginx] server index is ${SERVER_INDEX}"
   sed -i "s/index index.html index.htm/index ${SERVER_INDEX}/" $CONFIG_SITE
 fi
+
+if [[ $SERVER_GZIP_OPTIONS ]]
```

Comment:
discussed in real life. gzip is technically enabled by default in nginx, this is opening the gzip options to serve/process static content this way